### DISCUSSION TITLE
Pretty Cool.

### DISCUSSION THREAD

**Discussion by RalphHightower** [upptime/upptime#970] [General] [answered]
I just saw this project on my GitHub feed. I haven't tried it yet. I don't have any mission critical applications running on github.io, but I see where the utility comes in.

No problem to report. Here's a congratulatory **Atta boy, @AnandChowdhary!**

**Comment by AnandChowdhary** [comment, marked as answer]
Thank you! You can also use Upptime for tracking uptimes of any URLs, not necessarily those hosted on GitHub. So you person website or work-related notifications, all good use cases!
